app/csv.go

```diff
@@ -0,0 +1,25 @@
+package app
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+)
+
+func CSVReader() [][]string {
+
+	// open the file
+	csvFile, err := os.Open("../pokedex.csv")
+	if err != nil {
+		fmt.Println("error encountered opening csv file")
+		panic(err.Error())
```

Comment:
Same comment about panics, avoid them and make your function return both the value it should return plus the error